test/integration/controller/job/job_controller_test.go

```diff
@@ -275,6 +263,28 @@ var _ = ginkgo.Describe("Job controller for workloads with no queue set", func()
 			return k8sClient.Get(ctx, lookupKey, createdWorkload)
 		}, framework.Timeout, framework.Interval).Should(gomega.Succeed())
 	})
+
+	ginkgo.It("Should not reconcile suspend-in-creation jobs even queue is set", func() {
```

Comment:
What's the difference with the previous test?

Reply:
Seems useless, removed.